feature/bgp/policybase/otg_tests/aspath_community_test/README.md

```diff
@@ -0,0 +1,86 @@
+# RT-2.2: BGP Policy using AS Paths and Community sets
+
+## Summary
+
+BGP policy configuration for AS Paths and Community Sets
+
+## Procedure
+
+* Subtest 1
+    * Establish eBGP sessions between ATE port-1 and DUT port-1
+    * For IPv4 and IPv6 routes:
+        * Advertise IPv4 prefixes over IPv4 neighbor from ATE port-1, observe 
+            received prefixes at ATE port-2.
+    * Advertise IPv6 prefixes over IPv6 neighbor from ATE port-1, 
+        observe received prefixes at ATE port-2.
+    * Validate that traffic can be forwarded to **all** installed routes
+        between ATE port-1 and ATE port-2
+
+* Subtest 2 for as-path-set
+    * Create table based tests for each of the following policies
+    * Create an as-path-set/name "my_3_aspaths" with members and match options as follows
+        *  `{ as-path-set-member = [ "100", "200", "300" ], match-set-options=ANY }`
+    * Create an as-path-set/name "my_regex_aspaths" with one member as follows
+        * `{ as-path-set-member = [ "10[0-9]" ], match-set-options=ANY }`
+    * Create an as-path-set/name "all_3_aspaths" with members and match options as follows
+        * `{ as-path-set-member = [ "100", "200", "300" ], match-set-options=ALL }`
```

Comment:
There is no match-set-option leaf in as-path-set.
For as-path-set, the match-set-option is part of policy statement:
`/routing-policy/policy-definitions/policy-definition/statements/statement/conditions/bgp-conditions/match-as-path-set/config/match-set-options`

What is Right think.

Unfortunetly for community-set,  match-set-option is attribute of defined-set itself. What is inconsistency but also wrong way of modelig.

Reply:
Thank you for the useful comment! I corrected the syntax for match-set-options the as-path-set.  I wanted to be very explicit here because I expect to see questions, deviations and maybe some problems too.  The test will help us find those issues.

As far as changing the OC model for community-set, that is certainly possible, but it is a separate problem.  I started that [exploration](https://github.com/openconfig/models/commit/71b832f2e80298475cfb1e30b4c3617a23b23dbb#r128454643), but it could take time.   The tests can be refactored if/when the OC models are refactored.